src/draft-wullink-rpp-core.md

```diff
@@ -147,21 +147,45 @@ Some EPP result codes, like 01500, 02500, 02501 and 02502 are related to session
 
 When an error occurs that prevents processing of the requested action, an RPP server MUST respond using a Problem Detail
 document [@!RFC9457] detailing what went wrong, or what was not acceptable to the server.
-The `type` field SHOULD be a URN under the `urn:ietf:params:rpp:code:` namespace, using the RPP response code; e.g.
-with a header of `RPP-Code: 02005` the `type` field is `urn:ietf:params:rpp:code:02005`.
+The `type` field MUST be the `urn:ietf:params:rpp:problem` URN.
+The `status` field MUST reflect the HTTP status code.
+The document MUST contain an `errors` element, as a list of objects detailing individual errors.
+
+This document consists of the following fields:
+
+`type`
+: (required, string) This field SHOULD be a URN under the `urn:ietf:params:rpp:code:` namespace using the RPP response
+code; e.g. with a code of `02005` the `type` field is `urn:ietf:params:rpp:code:02005`.
 Implementations MAY use other URIs, for more specificity about custom error types.
 
-Implementations MAY add extension fields to the Problem Detail document to convey additional information about the
+`detail`
+: (required, string) A human-readable description of the error.
+
+`values`
+: (optional, list of objects) References to which values in the original request were not acceptable to the server.
+
+The `values` objects consist of the following fields:
+
+`path`
+: (required, string) The JSONPath [@!RFC9535] to the value referenced
+
+`reason`
+: (optional, string) A human-readable description of why the value was not acceptable
```

Comment:
this (and title) are human readable strings. So we have to do sth with the language.
EPP has "lang" tag on the level of each property.
We may also use Content-Language header and assume the language of human readable string is consistent across the whole response.